deploy-help/delete-database.sh

```diff
@@ -0,0 +1,43 @@
+#! /bin/bash -e
+
+cluster_name="$1"
+deployment_dir=~/deployment
+kubernetes_build_dir="$deployment_dir/concent-deployment/kubernetes/build"
+
+function run_kubernetes_script {
+    executed_script="$1"
+    cd $kubernetes_build_dir/
+    ./$executed_script
+}
+
+if [ -z "$cluster_name" ]; then
+    echo 'Please specify the name of database'
+    exit 1
+fi
```

Comment:
I'd just make it reset both databases. I can't think of many situation on the dev cluster where you would want to delete one but keep the other. It can lead to weird, inconsistent behavior.